spec.bs

```diff
@@ -358,19 +666,67 @@ A <dfn export>fenced frame config</dfn> is a struct with the following [=struct/
 <div algorithm>
   The <dfn method for=Fence>reportEvent(<var ignore>event</var>)</dfn> method steps are:
 
-  1. <span class=XXX>Fill this out!</span>
+  1. Let *instance* be this [=traversable navigable=]'s [=fenced frame config instance=].
+
+  1. If *instance* is null, then return.
+
+  1. If *instance*'s [=fenced frame config instance/mapped url=] is null, then the behavior is
+     unspecified.
+
+  1. If this [=document=]'s [=origin=] and *instance*'s
+     [=fenced frame config instance/mapped url=]'s origin are not [=same origin=], then return.
+
+  1. If *instance*'s [=fenced frame config instance/fenced frame reporter=] is null, then return.
```

Comment:
Please use `|instance|` syntax inside algorithms for variables here and elsewhere in this PR (as we do in the rest of the spec).